data_processing/common/constants.py

```diff
@@ -30,9 +30,18 @@
 APP_CFG				='APP_CFG'
 DATA_CFG			='DATA_CFG'
 SCHEMA_FILE			='data_ingestion_template_schema.yml'
-
 PUBLIC_DIR          ='/gpfs/mskmind_ess/mind_public'
 
+# Annotation API Data Mappings
+PROJECT_MAPPING = {'msk_mind_ov': "OV_16-158", "msk_mind_lung": "LUNG_18-193"}
```

Comment:
I think this is OK for now but we should think of a more configurable way of specifying this mapping. This amounts to hard coding.

Reply:
Right -- maybe a YAML here would work